recipes/resonance/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "resonance" %}
+{% set version = "0.2.0" %}
+{% set sha256 = "af554059984027a1419a5359339c0ecc1543edd9bdf107b07f6ae6722c51f063" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [py2k]
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - numpy >=1.13
+    - scipy >=0.19
+    - pandas >=0.20
+    - matplotlib >=2.0
+
+test:
+  imports:
+    - resonance
+    - resonance.version
+    - resonance.linear_systems
+    - resonance.functions
+    - resonance.tests
+
+about:
+  home: http://github.com/moorepants/resonance
+  license: CC-BY-4.0
+  summary: 'Learning mechanical vibrations through computation.'
+
+extra:
+  recipe-maintainers:
+    - moorepants
+    - ixjlyons
```

Comment:
@ixjlyons, are you ok with being added as a maintainer for this recipe?

Reply:
Yes, thanks for checking.